runtime/common/src/assigned_slots/mod.rs

```diff
@@ -193,6 +205,9 @@ pub mod pallet {
 			// We didn't return early above, so we didn't do anything.
 			Weight::zero()
 		}
+		fn on_runtime_upgrade() -> frame_support::weights::Weight {
+			migration::migrate_to_v2::<T>()
+		}
```

Comment:
We should add the migration in the runtime, like [these](https://github.com/paritytech/polkadot/blob/master/runtime/rococo/src/lib.rs#L1482)

Just as a good practice to follow, so it is easier to spot which migrations will get executed in the next upgrade.
This would also work, but the discoverability of this migration happening for other devs is worse.

Reply:
Done!